src/3d/symbols/qgspoint3dsymbol.cpp

```diff
@@ -66,6 +72,12 @@ void QgsPoint3DSymbol::readXml( const QDomElement &elem, const QgsReadWriteConte
 
   QDomElement elemTransform = elem.firstChildElement( QStringLiteral( "transform" ) );
   mTransform = Qgs3DUtils::stringToMatrix4x4( elemTransform.attribute( QStringLiteral( "matrix" ) ) );
+
+  QDomDocument doc( QStringLiteral( "symbol" ) );
+  doc.setContent( elem.attribute( QStringLiteral( "billboard-symbol" ) ) );
+  QDomElement symbolElem = doc.firstChildElement( QStringLiteral( "symbol" ) );
+
+  mBillboardSymbol = QgsSymbolLayerUtils::loadSymbol< QgsMarkerSymbol >( symbolElem, QgsReadWriteContext() );
```

Comment:
use "context" that is passed to readXml() method instead of creating an empty context - otherwise relative paths would not get resolved correctly to absolute paths